ledger/src/blockstore_meta.rs

```diff
@@ -225,10 +270,17 @@ impl SlotMeta {
     }
 
     pub(crate) fn new(slot: Slot, parent_slot: Option<Slot>) -> Self {
+        let connected_flags = if slot == 0 {
+            // Slot 0 is the start, mark it as having its' parent connected
+            // such that slot 0 becoming full will be updated as connected
+            ConnectedFlags::CONNECTED
```

Comment:
👍

Reply:
Good idea - added in https://github.com/solana-labs/solana/pull/29025